### DISCUSSION TITLE
Use esbuild additional entry points in Angular application builder

### DISCUSSION THREAD

**Discussion by ArielGueta** [nrwl/nx#23299] [Questions]
Hi!

I need to build a chrome extension using Angular and Nx and I need to have additional entry points for the extension files such as service worker, content script, etc. Is there anyway to pass [additional entry points](https://nx.dev/nx-api/esbuild/executors/esbuild#additionalentrypoints) to Angular application builder? if not, how can I solve this using Nx?

Btw, it'll be great if you'll have Nx plugin for chrome extension, the setup is complicated.

Thanks.

**Comment by Coly010** [comment]
The angular application builder does not offer a method for specifying additional entry points in the same manner as esbuild, however, you may be able to achieve what you're looking for with the `scripts` option: https://github.com/angular/angular-cli/blob/main/packages/angular/build/src/builders/application/schema.json#L40
